tests/test_requests.py

```diff
@@ -142,6 +146,15 @@ def test_participate_in_a_challenge_for_http_error_404(self):
         expected = "{}{}".format(self.expected.format(url), "\n")
         assert response == expected
 
+    @responses.activate
+    def test_display_your_submissions_for_http_error_404(self):
```

Comment:
Please change it & the following function names according to the current function.